### ISSUE TITLE
Rubocop Configuration - Phase 2

### ISSUE THREAD

**Issue by Adeynack** [nerdgeschoss/shimmer#42]
# Dependencies

- Don't start until #39 (Phase 1) is done

# Phase 2

- [ ] Relevant Cops for _RSpec_ are enabled explicitly for
  - [ ] [`-performance`](https://github.com/rubocop/rubocop-performance)
  - [ ] [`-rake`](https://github.com/rubocop/rubocop-rake)
  - [ ] [`-slim`](https://github.com/r7kamura/rubocop-slim)
  - [ ] [`-capybara`](https://github.com/rubocop/rubocop-capybara)
  - [ ] [`-thread_safety`](https://github.com/rubocop/rubocop-thread_safety)
  - [ ] Add up to `base`:
    - [ ] [`-bundler`](https://docs.rubocop.org/rubocop/1.50/cops_bundler.html)
    - [ ] [`-layout`](https://docs.rubocop.org/rubocop/1.50/cops_layout.html)
    - [ ] [`-lint`](https://docs.rubocop.org/rubocop/1.50/cops_lint.html)
    - [ ] [`-metrics`](https://docs.rubocop.org/rubocop/1.50/cops_metrics.html) (to consider)
    - [ ] [`-naming`](https://docs.rubocop.org/rubocop/1.50/cops_naming.html) (to consider)
    - [ ] [`-security`](https://docs.rubocop.org/rubocop/1.50/cops_security.html)
- [ ] _development-environment_ is, if needed, adjusted to use this configuration properly
- [ ] Following projects are referencing _Shimmer_'s rubocop and are stripped down of customized cops and have PRs open (waiting for this one) where the new styling is applied.
  - [ ] KS Core
  - [ ] Recordsale (??)
  - [ ] Flip Vinyl
  - [ ] Anymator
  - [ ] Copa Coupona ([PR](https://github.com/nerdgeschoss/copa-coupona/pull/364))
  - [ ] Tiny Farms
  - [ ] eaze
  - [ ] pull-request
  - [ ] Screamshot
  - [ ] app
  - [ ] accounting
  - [ ] LAIC (?)
- [ ] Document in _Shimmer_ how to base a project on the _Rubocop_ configuration.

**Comment by Adeynack** [comment]
I am closing this issue, as it seems no longer relevant (and outdated PR got closed).
